### ISSUE TITLE
How to instantiate a SepFormer object in external code

### ISSUE THREAD

**Issue by ethman** [speechbrain/speechbrain#499]
Hello SpeechBrain creators,

Thanks for putting together a comprehensive speech library. It's been very fun to explore! 

I am doing a project where I would like to train a SepFormer separation model within my own code base on my own data. It's not clear to me how I would instantiate the model outside of the SpeechBrain framework.

Ideally, what I would want is something like this:

    import speechbrain as sb
    
    my_hparams = { blah, blah, blah }
    sepformer = sb.SepFormer(**my_hparams)

Or this:

    import speechbrain as sb
    
    hparams_yaml = 'path/to/hparams/sepformer.yaml`
    sepformer = sb.SepFormer(hparams_yaml)

In an ideal world, after calling this constructor I would expect that the `sepformer` would be all set up, and I would be able to pass gradients through it, get output audio, and use my own data loaders that I already have written, just like as if I had built it from raw pytorch components outside of the SpeechBrain framework.

Is this currently possible? If so, how would I achieve this?

Thanks so much :D
